docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+
+Support for running on [Kubernetes](https://kubernetes.io/) is available in experimental status. The feature set is
+currently limited and not well-tested. This should not be used in production environments.
+
+## Setting Up Docker Images
+
+In order to run Spark on Kubernetes, a Docker image must be built and available on an accessible Docker registry. Spark
```

Comment:
I think it's okay for now. I'll discuss this and send a separate PR if we do need to change the wording and the command line args.